src/Tickets/Emails/Email/Ticket.php

```diff
@@ -176,6 +187,17 @@ public function get_subject(): string {
 		return $subject;
 	}
 
+	/**
+	 * Get default email subject.
+	 * 
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_subject() {
+		return esc_html__( 'Your tickets to {event-name}', 'event-tickets' );
```

Comment:
We'll probably have to work on placeholders and this one in particular, as the logic for Events will be coming from TEC. `{event_name}` is something that will be coming from TEC, hooked to the placeholders.

Reply:
Yeah. I figured we'd need to be doing something different with all the placeholders in the future. They also shouldn't be in any of the translation strings. I figured we'd leave that for a future ticket, though.